tests/unit/test_wheels.py

```diff
@@ -34,9 +35,31 @@ def test_build_and_upload_wheel():
             },
         )
 
-        wheels.upload_to_dbfs()
+        wheels.upload_to_dbfs(force_dependencies=False)
+        installation.assert_file_dbfs_uploaded(re.compile("wheels/databricks_labs_blueprint-*"))
+    for wheel in wheels._local_wheel:
+        assert not os.path.exists(wheel)
+
+
+def test_build_and_upload_wheel_with_no_dependency():
```

Comment:
also add an integration test to verify that things get uploaded to installation directory.

Reply:
Currently facing an issue with the test environment - to be completed